core/player/src/expressions/evaluator.ts

```diff
@@ -119,12 +131,21 @@ export class ExpressionEvaluator {
 
   private readonly defaultHookOptions: HookOptions;
 
-  public readonly operators = {
-    binary: new Map(Object.entries(DEFAULT_BINARY_OPERATORS)),
-    unary: new Map(Object.entries(DEFAULT_UNARY_OPERATORS)),
-    expressions: new Map<string, ExpressionHandler<any, any>>(
-      Object.entries(DEFAULT_EXPRESSION_HANDLERS),
+  public readonly operators: {
+    binary: Map<string, BinaryOperator>;
+    unary: Map<string, UnaryOperator>;
+    expressions: Map<string, ExpressionHandler<any, any>>;
+  } = {
+    binary: new Map<string, BinaryOperator>(
+      Object.entries(DEFAULT_BINARY_OPERATORS),
+    ),
+    unary: new Map<string, UnaryOperator>(
+      Object.entries(DEFAULT_UNARY_OPERATORS),
     ),
+    expressions: new Map<string, ExpressionHandler<any, any>>([
+      ...Object.entries(DEFAULT_EXPRESSION_HANDLERS),
+      ["await", DEFAULT_EXPRESSION_HANDLERS.waitFor],
```

Comment:
I know core Player is js based but I am hesitant to use the same keywords that javascript uses. On one hand it does make sense because it is in effect the same functionality in the same language but because core is supposed to be cross platform it just feels a bit weird.

Reply:
It's not just JS that uses the `await` keyword/handing -- Python, C#, (I think Dart might as well); though I get where your head is at.

Reply:
Good point, didn't know about Python and C#. If its a common enough keyword I think it should be fine.